samples/apply-scene-property-expressions/src/main/java/com/esri/arcgismaps/sample/applyscenepropertyexpressions/components/ApplyScenePropertyExpressionsViewModel.kt

```diff
@@ -0,0 +1,121 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.applyscenepropertyexpressions.components
+
+import android.app.Application
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableDoubleStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.Color
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.view.GraphicsOverlay
+import com.arcgismaps.mapping.view.SurfacePlacement
+import com.arcgismaps.mapping.symbology.SimpleMarkerSceneSymbol
+import com.arcgismaps.mapping.symbology.SimpleMarkerSceneSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.view.Camera
+import com.arcgismaps.mapping.view.Graphic
+import com.arcgismaps.toolkit.geoviewcompose.SceneViewProxy
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.launch
+
+class ApplyScenePropertyExpressionsViewModel(app: Application) : AndroidViewModel(app) {
+    // The ArcGISScene shown in the SceneView
+    var arcGISScene by mutableStateOf(
+        ArcGISScene(BasemapStyle.ArcGISImageryStandard).apply {
+            // Set initial viewpoint with a camera looking at the cone
+            val point = Point(x = 83.9, y = 28.4, z = 1000.0, spatialReference = SpatialReference.wgs84())
+            initialViewpoint = Viewpoint(
+                latitude = 0.0,
+                longitude = 0.0,
+                scale = 1.0,
+                camera = Camera(lookAtPoint = point, distance = 1000.0, heading = 0.0, pitch = 50.0, roll = 0.0)
```

Comment:
```suggestion
                camera = Camera(lookAtPoint = point, distance = 1000.0, heading = 0.0, pitch = 65.0, roll = 0.0)
```

This centers the graphic in the camera a bit better, on my device anyways. Not sure how the lookAtPoint interacts with the pitch honestly. The doc says "Creates a camera based on a point to look at, the distance to this point, heading, pitch, and roll. Note: the given values may mean that the point is not visible on screen. "